ro.fxdata.taskmonitor.viewer.appdata.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally, you should maintain this file in your GitLab repo.

Reply:
This file is part of the project: https://gitlab.com/taskmonitor/tkmviewer/-/blob/master/data/ro.fxdata.taskmonitor.viewer.appdata.xml.in . I'm merging the fixes we find here to my project repo.
Being the first time I submit a PR for flathub distribution is not clear to me from where this is sourced and noticed other people are placing it here. I assume the file here is overriding the one generated from the source code during build.

Reply:
I removed the file from this PR and update the content on the app distributed file: https://gitlab.com/taskmonitor/tkmviewer/-/blob/v1.0.0/data/ro.fxdata.taskmonitor.viewer.appdata.xml.in